jumpscale/sals/nettools/__init__.py

```diff
@@ -0,0 +1,457 @@
+# TODO: test *2
+
+import time
```

Comment:
please let's use j.data.time, it's has a very friend time api using arrow library

Reply:
it doesn't have sleep function